kcharles52/ah-node
# #165676805: server setup

**What does this PR do?**

- The pull request add a server setup to the project

**Description of Task to be completed?**

- add express framework
- create server
- add configurations for environments eg development, testing
- add linters
- setup babel transpiler

**How should this be manually tested?**

- Run the following commands in order
- fetch the branch with `git fetch ch-server-setup-165676805`
- checkout to the branch with `git checkout ch-server-setup-165676805`
- start the server with `npm run start`
- make a get request to the endpoint `http://localhost:400/api/data`
- you should get a json object `{
                                                          "message": "initial server setup"
                                                       }`

**Any background context you want to provide?**

- N/A

**What are the relevant pivotal tracker stories?**

- [#165676805](https://www.pivotaltracker.com/story/show/165676805)

**Screenshots (if appropriate)**

- N/A
